src/components/Column.vue

```diff
@@ -15,7 +15,7 @@
     <div v-if="!columnCollapsed(section?.gid ?? '')" @drop="onDrop($event, section?.gid ?? '')"
       @dragenter="onDragEnter($event)" @dragstart="onDragStart()" @dragend="onDragEnd()" @dragover.prevent
       class="droppable" v-bind:id="section?.gid">
-      <task v-for="task in tasks(section?.gid ?? '')" :task="task" :key="task.gid"></task>
+      <task v-for="task in tasks(section?.gid ?? '')" :singleTaskView="singleTaskView" :task="task" :key="task.gid" />
```

Comment:
Passing this is feels incorrect (see other comment).

I think this should probably be solved by flexbox styling